### ISSUE TITLE
Django 5.0 breakage

### ISSUE THREAD

**Issue by bobort** [SmileyChris/django-countries#447]
I unintentionally upgraded to Django 5.0 trying to fix some other issues in my project, and I came across this error when rendering a country field in a template:

`'BlankChoiceIterator' object has no attribute '__len__'

```
\Lib\site-packages\django_countries\widgets.py, line 29, in get_choices
class LazyChoicesMixin:
    def get_choices(self) -> ChoiceList:
        """
        When it's time to get the choices, if it was a lazy then figure it out
        now and memoize the result.
        """
        if isinstance(self._choices, Promise):
            self._choices: ChoiceList = list(self._choices)
                                            ^^^^^^^^^^^^^^^^^^^ …
        return self._choices
    def set_choices(self, value: ChoiceList):
        self._set_choices(value)
    choices = property(get_choices, set_choices)
```

**Comment by Deep-Chill** [comment]
Facing the same issue.

**Comment by washeck** [comment]
Fix is available but not merged: https://github.com/SmileyChris/django-countries/pull/438

**Comment by ml394** [comment]
So my local development environment is working, I removed the `list()` call from that line and it works for now, until the above fix is merged. 
```
self._choices: ChoiceList = self._choices
```
At least while I'm working locally this will hold over

**Comment by mapperfr** [comment]
Same issue here. I tried @ml394 workaround and it works, thanks!

**Comment by paulopaixaoamaral** [comment]
This is happening for me as well, can't get the Admin pages to work properly anymore. Would love for the fix to be merged soon 🙂

**Comment by maazahmed000** [comment]
same issue for me as well

**Comment by JulianTenschert** [comment]
Same issue here.

**Comment by eatsleepdeploy** [comment]
Same issue here also

**Comment by josepduarte** [comment]
Same issue for me.

**Comment by washeck** [comment]
@JulianTenschert @eatsleepdeploy @josepduarte and others: please stop spaming the issue with "Me too" comments. It's just filling everyone's inboxes and adding zero value. In the very first comment there is a thumb up emoticon you can use to express your interest in the issue.

**Comment by SmileyChris** [comment]
Fixed in the just-released v7.6 🥳

**Comment by pablobuenaposada** [comment]
thanks, seems to be working!
